src/stactools/sentinel2/product_metadata.py

```diff
@@ -123,13 +123,15 @@ def scene_id(self) -> str:
             )
         id_parts = self.product_id.split("_")
 
-        # Remove .SAFE
-        id_parts[-1] = id_parts[-1].replace(".SAFE", "")
+        sensor_id = id_parts[0]
```

Comment:
I think an example of what these values are expected to look like at each stage would be useful for trying to understand this code in the future.

Reply:
added example filenames in code